EasyPost/Rating.cs

```diff
@@ -1,32 +1,40 @@
-﻿using RestSharp;
+﻿// Rating.cs
+// Copyright (c) 2022 EasyPost
+// All rights reserved.
 
-using System;
 using System.Collections.Generic;
+using RestSharp;
+
+namespace EasyPost
+{
+    public class Rating : Resource
+    {
+        public List<CarrierAccount> carrier_accounts { get; set; }
 
-namespace EasyPost {
-    public class Rating : Resource {
-#pragma warning disable IDE1006 // Naming Styles
         public Address from_address { get; set; }
-        public Address to_address { get; set; }
+
         public List<Parcel> parcels { get; set; }
-        public List<CarrierAccount> carrier_accounts { get; set; }
-        public List<Object> ratings { get; set; }
-#pragma warning restore IDE1006 // Naming Styles
+
+        public List<object> ratings { get; set; }
+
+        public Address to_address { get; set; }
```

Comment:
Order is laid out in the `LayoutExample.cs` file. For these properties, it would be sort by non-static/static, then sub-sort by public/internal/protected/private, then sub-sort alphabetically.

Reply:
This is fantastic, thanks!